sympy/ntheory/primetest.py

```diff
@@ -110,6 +110,19 @@ def mr(n, bases):
     return True
 
 
+def isprimeE(n,b):
+	"""isprimeE(n) - Test whether n is prime or an Euler pseudoprime to base b."""
+	if (not mr(n,b)): return False
```

Comment:
Here and below, there should be a new line after  colon  (PEP8, you can check it online with http://pep8online.com/)

The parentheses around if conditions are not needed.

Reply:
Parenthesis are necessary for readability when writing the result of the condition and the case in one line

Reply:
I will change and add a new line after the colon according to PEP8.
thanks...